service/storage_service.cc

```diff
@@ -2079,7 +2080,7 @@ future<> storage_service::decommission() {
                 }).get();
                 slogger.info("decommission[{}]: Finished token ring movement, removing node={}, sync_nodes={}, ignore_nodes={}", uuid, endpoint, nodes, ignore_nodes);
             } catch (...) {
-                slogger.warn("decommission[{}]: Abort decommission operation started, removing node={}, sync_nodes={}, ignore_nodes={}", uuid, endpoint, nodes, ignore_nodes);
+                slogger.error("decommission[{}]: decommissioning node={}, sync_nodes={}, ignore_nodes={} failed: {}. Aborting operation", uuid, endpoint, nodes, ignore_nodes, std::current_exception());
```

Comment:
In fact this entire catch section looks like it is very similar across the different ops, maybe extract it into a function?

Reply:
Hmm, maybe this could work of each node op prints its own message, since they have slightly different parameters, but it's not a bad idea.